client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -707,7 +753,7 @@ class PatientSubmissionScreen extends React.Component {
         ),
         name: 'ClinicalInformation',
         values: {},
-        isComplete: this.isClinicalInformationComplete(),
+        isComplete: () => true,
```

Comment:
If the isClinicalInformationComplete() method is no longer used, let's get rid of it; however, is it safe to hardcode this to "true"?

Reply:
WIP